client/src/pages/ScanStore.tsx

```diff
@@ -0,0 +1,137 @@
+import React, {useEffect, useState} from 'react'
```

Comment:
It would be a great follow up tasks to start writing some initial tests for these pages!
Feel free to use a testing framework of your choice, some famous ones are enzyme, chai etc

Do check out snapshot testing as well:
https://jestjs.io/docs/en/snapshot-testing

Reply:
open issue for future work